tests/test_selection/test_information_value.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+
+from feature_engine.selection import SelectByInformationValue
+
+
+@pytest.mark.parametrize(
+    "_threshold", ["python", (True, False), [4.3, 3]]
+)
+def test_error_when_not_permitted_threshold(_threshold):
+    with pytest.raises(ValueError):
+        SelectByInformationValue(
+            variables=None,
+            threshold=_threshold,
+            ignore_format=False,
+            confirm_variables=False,
+        )
+
+
+def test_error_when_more_than_two_classes(df_enc_numeric):
```

Comment:
what is this testing?

Reply:
I understand the IV can only be used with binary classification. The test checks whether an error is raised when the dependent variable is multiclass.